src/containers/pages/InterventionPlan/UpdatePlan/index.tsx

```diff
@@ -138,10 +139,20 @@ const UpdatePlan = (props: RouteComponentProps<RouteParams> & UpdatePlanProps) =
     planStatus === PlanStatus.DRAFT &&
     isFIOrDynamicFI(initialValues.interventionType) &&
     initialValues.fiReason === CASE_TRIGGERED;
+
+  const isCaseTriggeredOrRoutinePlan =
+    CASE_TRIGGERED_DRAFT_EDIT_ADD_ACTIVITIES &&
+    (planStatus === PlanStatus.DRAFT ||
+      planStatus === PlanStatus.ACTIVE ||
+      planStatus === PlanStatus.COMPLETE) &&
+    isFIOrDynamicFI(initialValues.interventionType) &&
+    initialValues.fiReason !== CASE_TRIGGERED &&
+    initialValues.fiReason !== ROUTINE;
```

Comment:
Do we need this?
Also i think this condition will always be false. Right?

Reply:
Checks updated.